src/coreclr/src/jit/importer.cpp

```diff
@@ -4143,7 +4143,7 @@ GenTree* Compiler::impIntrinsic(GenTree*                newobjThis,
             case NI_System_MathF_FusedMultiplyAdd:
             {
 #ifdef TARGET_XARCH
-                if (compExactlyDependsOn(InstructionSet_FMA))
+                if (compExactlyDependsOn(InstructionSet_FMA) && supportSIMDTypes())
```

Comment:
This was an existing bug exposed by this PR.

Reply:
Not related to this change, but I wonder if this optimization can be implemented without requiring to support SIMD types

Reply:
Probably not easily and I'm not sure the distinction is worthwhile. ARM64 already requires `supportSIMDTypes` due to the ABI and Unix x64 will likely be in a similar boat due to its ABI.